Helper/Entity/ProductHelper.php

```diff
@@ -86,6 +89,26 @@ class ProductHelper
         'color',
     ];
 
+    /**
+     * ProductHelper constructor.
+     * 
```

Comment:
```suggestion
     *
```
@peterjaap Super sorry to ask you to change this again, but the tool is pretty strict :(